css/glpi.scss

```
@@ -34,6 +34,12 @@
 $enable-negative-margins: true;
 $link-hover-decoration: none;
 
-@import "includes/fonts";
-@import "includes/logos";
 @import "includes/includes";
+
+html:not([data-glpi-theme-dark="1"]) {
+    @import "../public/lib/tinymce/skins/ui/oxide/skin";
+}
+
+html[data-glpi-theme-dark="1"] {
+    @import "../public/lib/tinymce/skins/ui/oxide-dark/skin";
+}
```

You should add a rule in the webpack config to copy these files into `css/lib` (the same way it is done for tabler/bootstrap scss files).
```js
    {
        package: 'tinymce',
        from: 'skins/ui/oxide-*/skin.css',
        to: scssOutputPath,
    },
```

And then use the `~` prefix to import them.
```suggestion
html:not([data-glpi-theme-dark="1"]) {
    @import "~tinymce/skins/ui/oxide/skin";
}

html[data-glpi-theme-dark="1"] {
    @import "~tinymce/skins/ui/oxide-dark/skin";
}
```